src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -182,30 +215,48 @@ export default function UserManagersCollaborationEditTable({
           style={{ margin: '8px 16px', display: 'block' }}
         />
       ) : null}
-      {patchError && (
+      {revokeError && (
         <CustomAlert
           severity="error"
           titleId="COLLABORATION_REVOKE_ERROR"
-          onClose={() => {
-            onClearPatchError();
-          }}
+          onClose={onClearRevokeError}
         >
-          {patchError
-            ? patchError +
+          {revokeError
+            ? revokeError +
               '. ' +
               intl.formatMessage({
                 id: 'COLLAB_REVOKE_ERROR_SUPPLEMENTAL',
               })
             : intl.formatMessage({ id: 'UNKNOWN_ERROR' })}
         </CustomAlert>
       )}
-      {patchSuccess && (
+      {restoreError && (
+        <CustomAlert
+          severity="error"
+          titleId="COLLABORATION_REVOKE_ERROR"
```

Comment:
This should be "COLLABORATION_RESTORE_ERROR", or something similar.

Reply:
Resolved. Good catch.